src/model/model.js

```diff
@@ -487,6 +491,22 @@ export default class Model {
 	 * @param {module:engine/model/writer~Writer} writer `Writer` instance that has been used in the change block.
 	 */
 
+	/**
+	 * Fired when entering the first {@link module:engine/model/model~Model#enqueueChange} or
+	 * {@link module:engine/model/model~Model#change} block of the pending changes.
```

Comment:
That "pending changes" thing is misleading. It sounds like this is the Nth+ block. A one that was scheduled but needed to be postponed so it's "pending". Isn't it just the "outermost enqueueChange/change block"?